lib/l4p/tcp_rxtx.c

```diff
@@ -206,15 +206,16 @@ tcp_stream_adjust_tms(const struct tle_tcp_stream *s, uint32_t tms)
 }
 
 static inline void
-fill_tcph(struct rte_tcp_hdr *l4h, const struct tcb *tcb, union l4_ports port,
+fill_tcph(struct rte_tcp_hdr *l4h, const struct tle_tcp_stream *s, union l4_ports port,
 	uint32_t seq, uint8_t hlen, uint8_t flags)
 {
+	const struct tcb *tcb = &s->tcb;
```

Comment:
I may be missing something, but you've added a new argument...then here pulled out a part of it...then never use it again.

Should this line be here?

Reply:
It is used; just outside of the diff window. Look at (new) lines 222, 230, and 232.